lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
Sorry, I just re-read your comment...how very strange.  Can it be reproduced locally?